plugins/database/influxdb/influxdb.go

```diff
@@ -3,11 +3,11 @@ package influxdb
 import (
 	"context"
 	"fmt"
+	"github.com/hashicorp/vault/sdk/helper/template"
```

Comment:
You may not have set this up already, but the [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) tool is useful to organize imports in a consistent way. I suggest to try running that tool on this file! I would expect to see this import grouped with the other group (`"github.com/*"`) below.

This is just a stylistic suggestion. We usually try to organize imports consistently in Vault.